docs/DeveloperGuide.md

```diff
@@ -257,11 +257,26 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* Name: Kevin Bernard Long Zheng Wei
+* Age: 28
+* Gender: Male
+* Marital status: Single
+* Country: Singapore
+* Habits: Sleeping, Netflix, Gym, Gaming
+* Lifestyle: Code, eat, sleep, give part time tuition
+    * prefer desktop apps over other types
+    * can type fast
+    * prefers typing to mouse interactions
+    * is reasonably comfortable using CLI apps
+* Interest: Coding, building apps during his free time with friends and teaching people
+* Values: Work hard play hard
+* Salary: $12000/mth
+* Job: Full time A levels Mathematics tuition teacher (group and one on one) at his own new tuition company
```

Comment:
he manages a company and teaches so many tuitions at the same time? Is this a bit unrealistic?

Reply:
Ok will change them